src/views/RecordingPreviewer/RecordingPreviewer.tsx

```diff
@@ -10,19 +9,23 @@ import {
   getFileNameWithoutExtension,
 } from '@/utils/file'
 import { View } from '@/components/Layout/View'
-import { RequestsSection } from '@/views/Recorder/RequestsSection'
 import { createNewGeneratorFile } from '@/utils/generator'
 import { ProxyData } from '@/types'
 import { harToProxyData } from '@/utils/harToProxyData'
 import { getRoutePath } from '@/routeMap'
-import { Details } from '@/components/WebLogView/Details'
 import { useProxyDataGroups } from '@/hooks/useProxyDataGroups'
-import { EmptyMessage } from '@/components/EmptyMessage'
+import { RequestLog } from '../Recorder/RequestLog'
+import { useSettings } from '@/components/Settings/Settings.hooks'
+import { RecordingInspector } from '../Recorder/RecordingInspector'
+import { BrowserEvent } from '@/schemas/recording'
 
 export function RecordingPreviewer() {
+  const { data: settings } = useSettings()
+
   const [proxyData, setProxyData] = useState<ProxyData[]>([])
-  const [isLoading, setIsLoading] = useState(false)
-  const [selectedRequest, setSelectedRequest] = useState<ProxyData | null>(null)
+  const [browserEvents, setBrowserEvents] = useState<BrowserEvent[]>([])
+
+  const [isLoading, setIsLoading] = useState(true)
```

Comment:
We'll always be in a loading state when the component mounts, so this should be `true` by default.